tool_collections/samtools/.shed.yml

```
@@ -0,0 +1,12 @@
+auto_tool_repositories:
```

```suggestion
name: samtools
owner: iuc
description: Galaxy wrappers for the SAMtools package
homepage_url: http://www.htslib.org/
long_description: |
  SAM (Sequence Alignment/Map) format is a generic format for storing large
  nucleotide sequence alignments. The Samtools package of command line
  utilities lets you read, write, edit, index and view data in SAM format,
  and in the related binary formats BAM and CRAM.
remote_repository_url: https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/samtools
type: unrestricted
categories:
- SAM
auto_tool_repositories:
```

You need to add top-level information, including owner!